### PR TITLE
[ARTS-342] - Spring profile and label support

### PR DESCRIPTION
**Pull request by eladiw**
## Description
Profile and Label paramaters are now in the definition file and passed to the services (currently just the practitioner)

### Changes
- [ARTS-342] - added the missing attributes, parsed and passed to terraform

### Checklist:

- [x] Branch name follows convention (feature/arts-###-short-name OR fix/arts-###-short-name)
- [x] I have included a short-meaningful title, description and changes for this PR

[ARTS-342]: https://ndph-arts.atlassian.net/browse/ARTS-342

### CODE REVIEW

File: terraform/trial_rg/services.tf [hunk 1]
```diff
@@ -30,8 +30,9 @@ module "trial_app_service_practitioner" {
   # todo use private endpoint
   settings = {
     "SPRING_APPLICATION_NAME"     = "practitioner-service",
-    "SPRING_CLOUD_CONFIG_PROFILE" = "default",
-    "SPRING_CLOUD_CONFIG_LABEL"   = "main",
+    "SPRING_PROFILES_ACTIVE"      = var.sc_profile,
```
Comment (anatbal) [inline terraform/trial_rg/services.tf]: "SPRING_PROFILES_ACTIVE" - What does this var means? Why do we need both?

**Review by anatbal — COMMENTED** [review]
LGTM, asked a question, and also - sample_trial/services/practitioner-service/application-dev.yml- did you check the practitioner work with that change? Is the config server fetching both property files? (application + practitioner-service)

**Review by anatbal — DISMISSED** [review]

File: terraform/trial_rg/services.tf (hunk 1 above)
Comment (eladiw) [reply terraform/trial_rg/services.tf]: this one says which profile THIS service starts with.
and the second one says which config server profile to go to.
in other words:
when I set the active profile on the practitioner it affects the practitioner loaded files.
when I set the config profile it changes to which profile in thhe config server the practitioner asks for

File: .ci/trial-json-schema.json [hunk 2]
```diff
@@ -58,6 +58,12 @@
       "type" : "object",
       "additionalProperties" : false,
       "properties" : {
+        "profile" : {
+          "type" : "string"
+        },
+        "label" : {
+          "type" : "string"
+        },
```
Comment (tamirkamara) [inline .ci/trial-json-schema.json]: 1. Are those the right names to indicate what they will be used for?
2. Their location in the root of the spring_cloud element also indicates that they will be used across spring cloud services. Will they?

File: terraform/trial_rg/services.tf (hunk 1 above)
Comment (eladiw) [reply terraform/trial_rg/services.tf]: fixed. now only 1 var is used: SPRING_PROFILES_ACTIVE

File: .ci/trial-json-schema.json (hunk 2 above)
Comment (eladiw) [reply .ci/trial-json-schema.json]: as discussed f2f, moved to upper level. 
profile is now spring_profile as its affects both the properties of each service + affect which profile will be reach in the config server
label is now spring_config_label as it affects ONLY the branch of the config server

**Review by tamirkamara — DISMISSED** [review]

**Review by tamirkamara — APPROVED** [review]

**Review by anatbal — APPROVED** [review]

**Pull request merged**